src/main.js

```diff
@@ -106,9 +106,15 @@ var savedPosters = [];
 var currentPoster;
 
 // event listeners go here 👇
-
+window.addEventListener('load', setPoster);
```

Comment:
The syntax on the eventListener looks good. Naming of the function includes a verb and description - great!